lambda-multi-svc/README.md

```diff
@@ -24,6 +24,10 @@ First, make sure you have the AWS CLI installed and configured. Run the followin
 ```bash
 account_id=`aws sts get-caller-identity|jq -r ".Account"`
 ```
+OR
+```bash
+account_id=`aws sts get-caller-identity --query Account --output text`
```

Comment:
should we just replace the `jq` stuff with this?  Seems like it's fine to do

Reply:
yeah. removed the `jq` stuff.